packages/neutron-sdk/src/bindings/msg.rs

```diff
@@ -108,6 +108,8 @@ pub enum NeutronMsg {
         // Timeout timestamp in absolute nanoseconds since unix epoch.
         // The timeout is disabled when set to 0.
         timeout_timestamp: u64,
+        // Fees that will be used for different callbacks.
```

Comment:
Actually this comment doesn't clarify things (how exactly the fees are used for different callbacks).
Please, add a bit more info (and maybe a link to a relevant page in our documentation)